sources/launcher/Xenko.Launcher/Views/LauncherWindow.xaml

```diff
@@ -526,7 +526,8 @@
                             </Button>
                             <Border.ContextMenu>
                               <ContextMenu>
-                                <MenuItem Header="Show in Explorer" Command="{Binding ExploreCommand}" />
+                                <MenuItem Header="{x:Static r:Strings.ShowInExplorer}" Command="{Binding ExploreCommand}" />
```

Comment:
Ideally, this should use the translation system but it is not available for the launcher at the moment.
I don't remember the exact reason but I think it might be because of the way the launcher is packed as a single assembly/executable.